1001-2000/LIT1556Gorgor.xml

```diff
@@ -30,6 +31,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <witness corresp="PetermannIINachtrag50">BerlinRoLa Petermin II. Nachtr. 50</witness>
                     <!-- I will upload the Berlin MSS stubs very soon - ES -->
                 </listWit>
+                <listBibl type="secondary">
+                   <bibl><ptr target="bm:Six1999VOHD3"/>
+                        <citedRange unit="page">166</citedRange>                    
+                   </bibl>
+                  </listBibl>
+                <listBibl type="clavis">
+                   <bibl type="Graf1944GCALI">
```

Comment:
Graf is not in the list of claves included in the guidelines: https://betamasaheft.eu/Guidelines/?q=clavis&id=work-teiHeader. It can probably be encoded just as secondary

Reply:
yes, it must be secondary, since he unfortunately does not assign any clavis numbering, so he is not a clavis

Reply:
Ok, done.